init.lua

```diff
@@ -184,12 +184,14 @@ end
 --Match client
 local function match_client(c, forced_tags, hints)
     -- Don't prevent tags from being drag and dropped between screens
+    --[[
```

Comment:
Unrelated to the commit (will also make a lot of people angry due to the workflow change)